front/index.html

```diff
@@ -0,0 +1,202 @@
+<html>
+    <head>
+        <script src="https://unpkg.com/vue@next"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js "></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/locale/fr.min.js"></script>
+        <link rel="stylesheet" href="css/global.css">
+        <link rel="stylesheet" href="css/app.css">
+        <link rel="stylesheet" href="css/activity_bar.css">
+        <link rel="stylesheet" href="css/message_editor.css">
+        <link rel="stylesheet" href="css/room_list.css">
+        <link rel="stylesheet" href="css/room_editor.css">
+        <link rel="stylesheet" href="css/user-list.css">
+        <link rel="stylesheet" href="css/messages-list.css">
+        <link rel="stylesheet" href="css/login.css">
+    </head>
+    <body>
+        <div id="app">
+            <login v-on:login="onLogin($event)" v-bind:is-hidden="isConnected"></login>
+            <div class="app_vertical_layout">
```

Comment:
login is a popup, it could look weird if there was nothing behind.
And login already hides itself if not needed, that can be changed if needed though.

Reply:
Yes it the modal use case, see this : https://v3.vuejs.org/examples/modal.html#modal-component 

It mix v-if and close event, no need of props.

Reply:
so should i implement it this way ?
(and should i hide app_vertical_layout ?)

Reply:
Yes remove your is-hidden prop as it act the same way as a v-if (which is the way you do it in vueJS, it's built in for this)

Reply:
Fixed by fed9da7faf8faeafa7abff51775804715b62cf65